### ISSUE TITLE
[Bug]: Default action for applications is "Large Text"

### ISSUE THREAD

**Issue by torley-gallium** [quicksilver/Quicksilver#2960]
### Before submitting your bug report, please confirm you have completed the following steps

- [X] Searched the issue is not already listed in the [Issue List](https://github.com/quicksilver/Quicksilver/issues?q=is%3Aissue) (I've searched it)
- [X] Searched in the [Quicksilver Users' Group](https://groups.google.com/forum/#!forum/blacktree-quicksilver) for questions relating to tech support.
- [X] Downloaded the [latest version of Quicksilver](https://qsapp.com/) for your system and confirmed the issue still persists.

### Bug description

QuickSilver 2.4.1 is broken for me on macOS Ventura 13.5.

Every time I try to launch an app, it shows giant text.

Reverted back to 2.4.0, works fine.

### Steps to reproduce

1. Open QuickSilver with shortcut. (Using Bezel skin + Superfluous visual effects, no plugins.)
2. Select app to launch.

Then the directory path of the app appears in a gigantic font, but it doesn't launch.

### Expected behavior

It should launch the app and not freeze with giant text.

### MacOS Version

macOS 13

### Quicksilver Version

2.4.1

### Relevant Plugins

No plugins. Just customized some colors but using Bezel.

**Comment by corporate-gadfly** [comment]
Same here.

Defaults to `Large Type` always:
![image](https://github.com/quicksilver/Quicksilver/assets/1200537/727c3f5f-9532-4f41-b7d8-443f15dd58af)
Pressing `Tab` and then `O` will do the trick, but involves 2 extra clicks, every single time:
![image](https://github.com/quicksilver/Quicksilver/assets/1200537/707c2896-fe5a-4b5c-a76b-e33cb22b53c3)

**Comment by n8henrie** [comment]
@pjrobertson i wonder if this (and the other issue with shortcuts) could be related to the changes I made to get tests passing, in which I made sure that application paths were also recognized as valid objects for text actions.

If so, perhaps default action sorting can use `guessPrimaryType`?

Will need to look into this. On mobile now.

**Comment by torley-gallium** [comment]
@corporate-gadfly THANK YOU very much for the screenshots and substantiating that it's not just me.

I updated QuickSilver to 2.4.1 on a second Mac and it also repros.

@n8henrie I appreciate you looking into this!

**Comment by asmeurer** [comment]
For anyone seeing this, the workaround is to open the Quicksilver settings, go to General -> Actions, search for "large type" and uncheck it.

**Comment by DougHockin** [comment]
The "large type" fix worked for me as well. Sounds like another update is needed.

**Comment by n8henrie** [comment]
> For anyone seeing this, the workaround is to open the Quicksilver settings, go to General -> Actions, search for "large type" and uncheck it.

This just disables the `Large Type` action, which can be pretty useful (I use it all the time to make it easier to show text to bystanders) -- I recommend we find a better solution.

Unfortunately, I'm unable to reproduce the issue. Can you please try these steps?

1. Quit Quicksilver
1. Download `Quicksilver-debug.zip` from <https://github.com/quicksilver/Quicksilver/releases/tag/v2.4.1>
1. Temporarily hide all settings and plugins for testing: `mv ~/Library/Application\ Support/Quicksilver{,.bak}`
    - If not comfortable using the terminal, you can navigate in Finder to `~/Library/Application Support/` and manually rename `Quicksilver` to something like `Quicksilver.bak`
1. Open `Quicksilver-debug.zip`
    1. Unzip the file
    2. Right click (or ctrl click) `Quicksilver.app` and click `Open` (double clicking won't work)
    3. It will give you a warning the first time and refuse to open. Do the same thing a second time and it should work.
1. Type `Preview`
1. Mine defaults to `Open`. If I type <kbd>tab</kbd><kbd>l</kbd> it then goes to `Large Type`
1. When done testing, quit `Quicksilver-debug`  and don't forget to `mv ~/Library/Application\ Support/Quicksilver{.bak,}` (or otherwise reverse the renaming from above)

**Comment by asmeurer** [comment]
I can reproduce the problem in the debug build.

**Comment by n8henrie** [comment]
Of course -- the debug build is essentially unchanged from the version used to open the issue. The question is whether the issue persists after quitting QS, deleting `~/Library/Caches/Quicksilver`, and "hiding" `~/Library/Application Support/Quicksilver`

Thanks for getting back to me quickly -- can you confirm that you also took the other steps?

**Comment by corporate-gadfly** [comment]
@n8henrie : I hear you. I use `Large Type` all the time, but the hit of not being able to open *every single time* was just too much. Hence the work around of disabling it.

I must admit, I'm guilty of not following your debug procedure, but, IMO, all your instructions prove is that this scenario doesn't happen with a clean install of `2.4.1`. A few of us, who have existing `2.4.0`, under certain conditions _(or all?)_, when we ugpraded to `2.4.1`, the bug happened automatically without doing anything else.

**Comment by asmeurer** [comment]
I did delete the caches, but I didn't see the bit about hiding the application support folder. Let me try again.

**Comment by asmeurer** [comment]
OK, I don't see the issue when I hide Application Support/Quicksilver.

**Comment by n8henrie** [comment]
So it sounds like the issue possibly has to do with your local configuration or action ranking, possibly something in `~/Library/Application Support/Quicksilver/Actions.plist`?

Can you try restoring your usual app support file but temporarily renaming just that file? (Should probably delete the cache before each trial run regardless).

**Comment by asmeurer** [comment]
Renaming Actions.plist makes the issue go away. Would it help if I uploaded my Actions.plist?

**Comment by corporate-gadfly** [comment]
> Renaming Actions.plist makes the issue go away. Would it help if I uploaded my Actions.plist?

I can confirm the same.

The original file that I backed up and the one that got created at next run are radically different:
```
wc -l Actions-2023-09-13.plist Actions.plist
 1720 Actions-2023-09-13.plist
 1114 Actions.plist
```

**Comment by n8henrie** [comment]
I'm not sure that uploading your actions plist is necessary, thanks for the offer, I'll let you know if that changes.

I think the issue is that applications are now seen as eligible targets for string actions; for example, one can select `Safari.app` and then run `Large Type` and see `/System/Volumes/Preboot/Cryptexes/App/System/Applications/Safari.app` in large text, just like one can enter `Hello there!` or any other string and show that in large type.

The issue is that -- for whatever reason -- `Large Type` has become the *default* action on your system.

**Comment by corporate-gadfly** [comment]
IIRC, for `Large Type` to become automatically eligible for the action, the text had to have a `.`. If it didn't, then you could still make it the action by tabbing over. Do I recall correctly?

Following that train of thought, is `/` the new `.`?

**Comment by asmeurer** [comment]
"Large type" is rank 24 in the settings. Presumably something is overriding that ranking.

**Comment by asmeurer** [comment]
Oh I just noticed that "open/reveal" is rank 25. I had thought that the rank 1 "run/open" was the open action. If I move that above "large type" it also fixes the issue.

**Comment by corporate-gadfly** [comment]
Naive question. What would I lose by removing the original `Actions.plist`? Actions that were built up over the years by installing other software?

**Comment by n8henrie** [comment]
I think you'd lose your history (for example a specific catalog item that you always want to run a specific action for).

**Comment by aberkvam** [comment]
Just a wild guess, but this commit removes ".app" from applications.  Maybe that affects another test down the road?

https://github.com/quicksilver/Quicksilver/commit/260eee39000984b5f93ed15c0781ab7cba98204e

**Comment by n8henrie** [comment]
Thanks for the thought! That commit sets the label, which I think was previously empty / unset for applications (and why the test was previously failing).

**Comment by n8henrie** [comment]
@corporate-gadfly -- would you be willing to share your `actions.plist`? I'd like to compare with my own. I don't *think* there would usually be anything too private in `actions.plist`, but it's a little long to be sure with a glance through, so I hesitate to ask that you post it openly in the thread (though that's certainly an option). Alternatively you could email it to me; you can find my personal email at https://n8henrie.com/contact/ .

**Comment by corporate-gadfly** [comment]
oh sorry @n8henrie for the delay.

I'm afraid that won't be possible. I deleted the file a couple of weeks ago (sorry about that) and haven't missed it one single bit.

**Comment by aberkvam** [comment]
I believe I'm having the same issue (although Quicksilver keeps trying to "Calculate" apps for me, not "Large Text").  I haven't done any workarounds yet so I'm still experiencing the issue.  Here's my Actions.plist file:

`<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>actionActivation</key>
	<dict>
		<key>AppActivateAction</key>
		<false/>
		<key>AppHideAction</key>
		<false/>
		<key>AppHideOthersAction</key>
		<false/>
		<key>AppOpenFileAction</key>
		<true/>
		<key>AppQuitAction</key>
		<true/>
		<key>AppQuitOthersAction</key>
		<false/>
		<key>AppRelaunchAction</key>
		<true/>
		<key>AppShowHideAction</key>
		<false/>
		<key>AppleScriptOpenFilesAction</key>
		<true/>
		<key>AppleScriptProcessTextAction</key>
		<true/>
		<key>AppleScriptRunAction</key>
		<true/>
		<key>AppleScriptRunTextAction</key>
		<true/>
		<key>CalculatorCalculateFormulaAction</key>
		<true/>
		<key>DiskEjectAction</key>
		<true/>
		<key>FileAlwaysOpenTypeWithAction</key>
		<true/>
		<key>FileCopyToAction</key>
		<true/>
		<key>FileDeleteAction</key>
		<false/>
		<key>FileGetHFSPathAction</key>
		<true/>
		<key>FileGetInfoAction</key>
		<true/>
		<key>FileGetPathAction</key>
		<false/>
		<key>FileGetPosixPathAction</key>
		<false/>
		<key>FileGetURLAction</key>
		<false/>
		<key>FileMakeAliasInAction</key>
		<true/>
		<key>FileMakeHardLinkInAction</key>
		<false/>
		<key>FileMakeLinkInAction</key>
		<false/>
		<key>FileMoveToAction</key>
		<true/>
		<key>FileOpenAction</key>
		<true/>
		<key>FileOpenWithAction</key>
		<true/>
		<key>FileRenameAction</key>
		<true/>
		<key>FileRevealAction</key>
		<true/>
		<key>FileToTrashAction</key>
		<true/>
		<key>PasteboardCopyAction</key>
		<true/>
		<key>PasteboardPasteAction</key>
		<true/>
		<key>PasteboardPasteActionAsPlainText</key>
		<true/>
		<key>QSABContactAddDataAction</key>
		<true/>
		<key>QSABContactEditAction</key>
		<true/>
		<key>QSABContactShowAction</key>
		<true/>
		<key>QSCatalogAddEntryAction</key>
		<false/>
		<key>QSCatalogEntryRescanAction</key>
		<true/>
		<key>QSCatalogEntryShowAction</key>
		<true/>
		<key>QSCommandAddTriggerAction</key>
		<true/>
		<key>QSCommandExecuteAction</key>
		<true/>
		<key>QSCommandExecuteAfterDelayAction</key>
		<true/>
		<key>QSCommandExecuteAtTimeAction</key>
		<true/>
		<key>QSCommandSaveAction</key>
		<true/>
		<key>QSComposeEmailItemAction</key>
		<true/>
		<key>QSComposeEmailItemReverseAction</key>
		<true/>
		<key>QSCreateFileAction</key>
		<true/>
		<key>QSCreateFileTemplate</key>
		<true/>
		<key>QSDirectEmailItemAction</key>
		<false/>
		<key>QSDirectEmailItemReverseAction</key>
		<false/>
		<key>QSEmailAction</key>
		<true/>
		<key>QSEmailItemAction</key>
		<false/>
		<key>QSEmailItemReverseAction</key>
		<false/>
		<key>QSLargeTypeAction</key>
		<true/>
		<key>QSLoginItemAddAction</key>
		<false/>
		<key>QSLoginItemRemoveAction</key>
		<false/>
		<key>QSNewFolderAction</key>
		<true/>
		<key>QSObjCSendMessageAction</key>
		<true/>
		<key>QSObjectAssignMnemonic</key>
		<true/>
		<key>QSObjectExplodeCombinedAction</key>
		<true/>
		<key>QSObjectLogConsoleAction</key>
		<false/>
		<key>QSObjectSearchChildrenAction</key>
		<true/>
		<key>QSObjectSelectAction</key>
		<false/>
		<key>QSObjectShowActionMenu</key>
		<true/>
		<key>QSObjectShowChildMenu</key>
		<true/>
		<key>QSObjectShowChildrenAction</key>
		<true/>
		<key>QSObjectShowMenu</key>
		<true/>
		<key>QSObjectShowSourceAction</key>
		<true/>
		<key>QSSafariAddToReadingListAction</key>
		<true/>
		<key>QSTextShowDialogAction</key>
		<false/>
		<key>QSTextSpeakAction</key>
		<false/>
		<key>QSTextTypeAction</key>
		<true/>
		<key>QSiTunesAddToPlaylistAction</key>
		<true/>
		<key>QSiTunesDislikeAction</key>
		<true/>
		<key>QSiTunesDownloadAction</key>
		<true/>
		<key>QSiTunesGetLyrics</key>
		<true/>
		<key>QSiTunesLoveAction</key>
		<true/>
		<key>QSiTunesOpenBookletAction</key>
		<true/>
		<key>QSiTunesPlayItemAction</key>
		<true/>
		<key>QSiTunesRevealItem</key>
		<true/>
		<key>QSiTunesSelectAirPlayDevices</key>
		<true/>
		<key>QSiTunesSelectEQPreset</key>
		<true/>
		<key>QSiTunesToggleEnabled</key>
		<true/>
		<key>QSiTunesToggleShuffle</key>
		<true/>
		<key>URLEmailAction</key>
		<true/>
		<key>URLJSAction</key>
		<true/>
		<key>URLOpenAction</key>
		<true/>
		<key>URLOpenActionInBackground</key>
		<true/>
		<key>URLOpenWithAction</key>
		<false/>
	</dict>
	<key>actionIndirects</key>
	<dict/>
	<key>actionMenuActivation</key>
	<dict>
		<key>AppActivateAction</key>
		<false/>
		<key>AppHideAction</key>
		<false/>
		<key>AppHideOthersAction</key>
		<false/>
		<key>AppOpenFileAction</key>
		<true/>
		<key>AppQuitAction</key>
		<true/>
		<key>AppQuitOthersAction</key>
		<false/>
		<key>AppRelaunchAction</key>
		<true/>
		<key>AppShowHideAction</key>
		<false/>
		<key>AppleScriptOpenFilesAction</key>
		<true/>
		<key>AppleScriptProcessTextAction</key>
		<true/>
		<key>AppleScriptRunAction</key>
		<true/>
		<key>AppleScriptRunTextAction</key>
		<true/>
		<key>CalculatorCalculateFormulaAction</key>
		<true/>
		<key>DiskEjectAction</key>
		<true/>
		<key>FileAlwaysOpenTypeWithAction</key>
		<true/>
		<key>FileCopyToAction</key>
		<true/>
		<key>FileDeleteAction</key>
		<false/>
		<key>FileGetHFSPathAction</key>
		<true/>
		<key>FileGetInfoAction</key>
		<true/>
		<key>FileGetPathAction</key>
		<false/>
		<key>FileGetPosixPathAction</key>
		<false/>
		<key>FileGetURLAction</key>
		<false/>
		<key>FileMakeAliasInAction</key>
		<true/>
		<key>FileMakeHardLinkInAction</key>
		<false/>
		<key>FileMakeLinkInAction</key>
		<false/>
		<key>FileMoveToAction</key>
		<true/>
		<key>FileOpenAction</key>
		<true/>
		<key>FileOpenWithAction</key>
		<true/>
		<key>FileRenameAction</key>
		<true/>
		<key>FileRevealAction</key>
		<true/>
		<key>FileToTrashAction</key>
		<true/>
		<key>PasteboardCopyAction</key>
		<true/>
		<key>PasteboardPasteAction</key>
		<true/>
		<key>PasteboardPasteActionAsPlainText</key>
		<true/>
		<key>QSABContactAddDataAction</key>
		<true/>
		<key>QSABContactEditAction</key>
		<true/>
		<key>QSABContactShowAction</key>
		<true/>
		<key>QSCatalogAddEntryAction</key>
		<false/>
		<key>QSCatalogEntryRescanAction</key>
		<true/>
		<key>QSCatalogEntryShowAction</key>
		<true/>
		<key>QSCommandAddTriggerAction</key>
		<true/>
		<key>QSCommandExecuteAction</key>
		<true/>
		<key>QSCommandExecuteAfterDelayAction</key>
		<true/>
		<key>QSCommandExecuteAtTimeAction</key>
		<true/>
		<key>QSCommandSaveAction</key>
		<true/>
		<key>QSComposeEmailItemAction</key>
		<true/>
		<key>QSComposeEmailItemReverseAction</key>
		<true/>
		<key>QSCreateFileAction</key>
		<true/>
		<key>QSCreateFileTemplate</key>
		<true/>
		<key>QSDirectEmailItemAction</key>
		<false/>
		<key>QSDirectEmailItemReverseAction</key>
		<false/>
		<key>QSEmailAction</key>
		<true/>
		<key>QSEmailItemAction</key>
		<false/>
		<key>QSEmailItemReverseAction</key>
		<false/>
		<key>QSLargeTypeAction</key>
		<true/>
		<key>QSLoginItemAddAction</key>
		<false/>
		<key>QSLoginItemRemoveAction</key>
		<false/>
		<key>QSNewFolderAction</key>
		<true/>
		<key>QSObjCSendMessageAction</key>
		<true/>
		<key>QSObjectAssignMnemonic</key>
		<true/>
		<key>QSObjectExplodeCombinedAction</key>
		<true/>
		<key>QSObjectLogConsoleAction</key>
		<false/>
		<key>QSObjectSearchChildrenAction</key>
		<true/>
		<key>QSObjectSelectAction</key>
		<false/>
		<key>QSObjectShowActionMenu</key>
		<true/>
		<key>QSObjectShowChildMenu</key>
		<true/>
		<key>QSObjectShowChildrenAction</key>
		<true/>
		<key>QSObjectShowMenu</key>
		<true/>
		<key>QSObjectShowSourceAction</key>
		<true/>
		<key>QSSafariAddToReadingListAction</key>
		<true/>
		<key>QSTextShowDialogAction</key>
		<false/>
		<key>QSTextSpeakAction</key>
		<false/>
		<key>QSTextTypeAction</key>
		<true/>
		<key>QSiTunesAddToPlaylistAction</key>
		<true/>
		<key>QSiTunesDislikeAction</key>
		<true/>
		<key>QSiTunesDownloadAction</key>
		<true/>
		<key>QSiTunesGetLyrics</key>
		<true/>
		<key>QSiTunesLoveAction</key>
		<true/>
		<key>QSiTunesOpenBookletAction</key>
		<true/>
		<key>QSiTunesPlayItemAction</key>
		<true/>
		<key>QSiTunesRevealItem</key>
		<true/>
		<key>QSiTunesSelectAirPlayDevices</key>
		<true/>
		<key>QSiTunesSelectEQPreset</key>
		<true/>
		<key>QSiTunesToggleEnabled</key>
		<true/>
		<key>QSiTunesToggleShuffle</key>
		<true/>
		<key>URLEmailAction</key>
		<true/>
		<key>URLJSAction</key>
		<true/>
		<key>URLOpenAction</key>
		<true/>
		<key>URLOpenActionInBackground</key>
		<true/>
		<key>URLOpenWithAction</key>
		<false/>
	</dict>
	<key>actionNames</key>
	<dict/>
	<key>actionPrecedence</key>
	<dict>
		<key>AppActivateAction</key>
		<real>0.0</real>
		<key>AppHideAction</key>
		<real>0.0</real>
		<key>AppHideOthersAction</key>
		<real>0.0</real>
		<key>AppOpenFileAction</key>
		<real>0.0</real>
		<key>AppQuitAction</key>
		<real>0.0</real>
		<key>AppQuitOthersAction</key>
		<real>0.0</real>
		<key>AppRelaunchAction</key>
		<real>0.0</real>
		<key>AppShowHideAction</key>
		<real>0.0</real>
		<key>AppleScriptOpenFilesAction</key>
		<real>4</real>
		<key>AppleScriptProcessTextAction</key>
		<real>4</real>
		<key>AppleScriptRunAction</key>
		<real>4</real>
		<key>AppleScriptRunTextAction</key>
		<real>0.0</real>
		<key>CalculatorCalculateFormulaAction</key>
		<real>5</real>
		<key>DiskEjectAction</key>
		<real>0.0</real>
		<key>FileAlwaysOpenTypeWithAction</key>
		<real>0.10000000000000001</real>
		<key>FileCopyToAction</key>
		<real>0.0</real>
		<key>FileDeleteAction</key>
		<real>0.0</real>
		<key>FileGetHFSPathAction</key>
		<real>-0.5</real>
		<key>FileGetInfoAction</key>
		<real>0.0</real>
		<key>FileGetPathAction</key>
		<real>-0.5</real>
		<key>FileGetPosixPathAction</key>
		<real>-0.5</real>
		<key>FileGetURLAction</key>
		<real>-0.5</real>
		<key>FileMakeAliasInAction</key>
		<real>-0.5</real>
		<key>FileMakeHardLinkInAction</key>
		<real>-0.5</real>
		<key>FileMakeLinkInAction</key>
		<real>-0.5</real>
		<key>FileMoveToAction</key>
		<real>0.0</real>
		<key>FileOpenAction</key>
		<real>1</real>
		<key>FileOpenWithAction</key>
		<real>0.10000000000000001</real>
		<key>FileRenameAction</key>
		<real>0.0</real>
		<key>FileRevealAction</key>
		<real>0.10000000149011611</real>
		<key>FileToTrashAction</key>
		<real>0.0</real>
		<key>PasteboardCopyAction</key>
		<real>-0.5</real>
		<key>PasteboardPasteAction</key>
		<real>-0.5</real>
		<key>PasteboardPasteActionAsPlainText</key>
		<real>-0.5</real>
		<key>QSABContactAddDataAction</key>
		<real>0.0</real>
		<key>QSABContactEditAction</key>
		<real>0.0</real>
		<key>QSABContactShowAction</key>
		<real>3</real>
		<key>QSCatalogAddEntryAction</key>
		<real>-0.050000000000000003</real>
		<key>QSCatalogEntryRescanAction</key>
		<real>2</real>
		<key>QSCatalogEntryShowAction</key>
		<real>3</real>
		<key>QSCommandAddTriggerAction</key>
		<real>0.0</real>
		<key>QSCommandExecuteAction</key>
		<real>3</real>
		<key>QSCommandExecuteAfterDelayAction</key>
		<real>2</real>
		<key>QSCommandExecuteAtTimeAction</key>
		<real>2</real>
		<key>QSCommandSaveAction</key>
		<real>2</real>
		<key>QSComposeEmailItemAction</key>
		<real>0.0</real>
		<key>QSComposeEmailItemReverseAction</key>
		<real>0.0</real>
		<key>QSCreateFileAction</key>
		<real>0.0</real>
		<key>QSCreateFileTemplate</key>
		<real>0.0</real>
		<key>QSDirectEmailItemAction</key>
		<real>0.0</real>
		<key>QSDirectEmailItemReverseAction</key>
		<real>0.0</real>
		<key>QSEmailAction</key>
		<real>0.0</real>
		<key>QSEmailItemAction</key>
		<real>0.0</real>
		<key>QSEmailItemReverseAction</key>
		<real>0.0</real>
		<key>QSLargeTypeAction</key>
		<real>1</real>
		<key>QSLoginItemAddAction</key>
		<real>0.0</real>
		<key>QSLoginItemRemoveAction</key>
		<real>0.0</real>
		<key>QSNewFolderAction</key>
		<real>0.0</real>
		<key>QSObjCSendMessageAction</key>
		<real>5</real>
		<key>QSObjectAssignMnemonic</key>
		<real>-0.20000000000000001</real>
		<key>QSObjectExplodeCombinedAction</key>
		<real>0.0</real>
		<key>QSObjectLogConsoleAction</key>
		<real>0.0</real>
		<key>QSObjectSearchChildrenAction</key>
		<real>0.0</real>
		<key>QSObjectSelectAction</key>
		<real>0.0</real>
		<key>QSObjectShowActionMenu</key>
		<real>0.0</real>
		<key>QSObjectShowChildMenu</key>
		<real>0.0</real>
		<key>QSObjectShowChildrenAction</key>
		<real>0.0</real>
		<key>QSObjectShowMenu</key>
		<real>0.0</real>
		<key>QSObjectShowSourceAction</key>
		<real>0.0</real>
		<key>QSSafariAddToReadingListAction</key>
		<real>0.0</real>
		<key>QSTextShowDialogAction</key>
		<real>0.0</real>
		<key>QSTextSpeakAction</key>
		<real>0.0</real>
		<key>QSTextTypeAction</key>
		<real>0.0</real>
		<key>QSiTunesAddToPlaylistAction</key>
		<real>1</real>
		<key>QSiTunesDislikeAction</key>
		<real>0.0</real>
		<key>QSiTunesDownloadAction</key>
		<real>0.0</real>
		<key>QSiTunesGetLyrics</key>
		<real>0.0</real>
		<key>QSiTunesLoveAction</key>
		<real>0.0</real>
		<key>QSiTunesOpenBookletAction</key>
		<real>0.0</real>
		<key>QSiTunesPlayItemAction</key>
		<real>3</real>
		<key>QSiTunesRevealItem</key>
		<real>1</real>
		<key>QSiTunesSelectAirPlayDevices</key>
		<real>1</real>
		<key>QSiTunesSelectEQPreset</key>
		<real>2</real>
		<key>QSiTunesToggleEnabled</key>
		<real>0.0</real>
		<key>QSiTunesToggleShuffle</key>
		<real>0.0</real>
		<key>URLEmailAction</key>
		<real>0.0</real>
		<key>URLJSAction</key>
		<real>0.0</real>
		<key>URLOpenAction</key>
		<real>3</real>
		<key>URLOpenActionInBackground</key>
		<real>1</real>
		<key>URLOpenWithAction</key>
		<real>0.0</real>
	</dict>
	<key>actionRanking</key>
	<array>
		<string>QSObjCSendMessageAction</string>
		<string>CalculatorCalculateFormulaAction</string>
		<string>AppleScriptProcessTextAction</string>
		<string>AppleScriptOpenFilesAction</string>
		<string>AppleScriptRunAction</string>
		<string>URLOpenAction</string>
		<string>QSCatalogEntryShowAction</string>
		<string>QSCommandExecuteAction</string>
		<string>QSiTunesPlayItemAction</string>
		<string>QSABContactShowAction</string>
		<string>QSCommandExecuteAfterDelayAction</string>
		<string>QSCommandExecuteAtTimeAction</string>
		<string>QSCommandSaveAction</string>
		<string>QSCatalogEntryRescanAction</string>
		<string>QSiTunesSelectEQPreset</string>
		<string>QSLargeTypeAction</string>
		<string>FileOpenAction</string>
		<string>URLOpenActionInBackground</string>
		<string>QSiTunesRevealItem</string>
		<string>QSiTunesAddToPlaylistAction</string>
		<string>QSiTunesSelectAirPlayDevices</string>
		<string>FileRevealAction</string>
		<string>FileAlwaysOpenTypeWithAction</string>
		<string>FileOpenWithAction</string>
		<string>FileToTrashAction</string>
		<string>QSCommandAddTriggerAction</string>
		<string>AppActivateAction</string>
		<string>FileGetInfoAction</string>
		<string>QSCreateFileAction</string>
		<string>QSObjectShowChildrenAction</string>
		<string>QSTextShowDialogAction</string>
		<string>URLOpenWithAction</string>
		<string>FileMoveToAction</string>
		<string>AppShowHideAction</string>
		<string>QSObjectShowChildMenu</string>
		<string>FileRenameAction</string>
		<string>QSCreateFileTemplate</string>
		<string>URLJSAction</string>
		<string>QSObjectSearchChildrenAction</string>
		<string>AppOpenFileAction</string>
		<string>AppHideOthersAction</string>
		<string>QSLoginItemRemoveAction</string>
		<string>QSObjectShowMenu</string>
		<string>QSObjectShowSourceAction</string>
		<string>QSNewFolderAction</string>
		<string>URLEmailAction</string>
		<string>QSTextSpeakAction</string>
		<string>FileCopyToAction</string>
		<string>DiskEjectAction</string>
		<string>QSObjectShowActionMenu</string>
		<string>QSLoginItemAddAction</string>
		<string>AppQuitAction</string>
		<string>FileDeleteAction</string>
		<string>QSTextTypeAction</string>
		<string>AppleScriptRunTextAction</string>
		<string>AppHideAction</string>
		<string>AppQuitOthersAction</string>
		<string>AppRelaunchAction</string>
		<string>QSObjectSelectAction</string>
		<string>QSiTunesOpenBookletAction</string>
		<string>QSiTunesToggleEnabled</string>
		<string>QSiTunesToggleShuffle</string>
		<string>QSiTunesGetLyrics</string>
		<string>QSSafariAddToReadingListAction</string>
		<string>QSABContactEditAction</string>
		<string>QSABContactAddDataAction</string>
		<string>QSEmailItemAction</string>
		<string>QSComposeEmailItemAction</string>
		<string>QSDirectEmailItemReverseAction</string>
		<string>QSEmailAction</string>
		<string>QSEmailItemReverseAction</string>
		<string>QSDirectEmailItemAction</string>
		<string>QSComposeEmailItemReverseAction</string>
		<string>QSObjectExplodeCombinedAction</string>
		<string>QSiTunesDownloadAction</string>
		<string>QSiTunesDislikeAction</string>
		<string>QSiTunesLoveAction</string>
		<string>QSObjectLogConsoleAction</string>
		<string>QSCatalogAddEntryAction</string>
		<string>QSObjectAssignMnemonic</string>
		<string>PasteboardPasteActionAsPlainText</string>
		<string>PasteboardCopyAction</string>
		<string>FileMakeLinkInAction</string>
		<string>FileMakeAliasInAction</string>
		<string>FileGetURLAction</string>
		<string>FileGetHFSPathAction</string>
		<string>PasteboardPasteAction</string>
		<string>FileGetPathAction</string>
		<string>FileGetPosixPathAction</string>
		<string>FileMakeHardLinkInAction</string>
	</array>
</dict>
</plist>`

**Comment by n8henrie** [comment]
Thanks @aberkvam, I'll take a closer look soon!

@corporate-gadfly @torley-gallium Would you by change try running the debug version from [here](https://github.com/quicksilver/Quicksilver/actions/runs/6363050382) and see whether or not the unwanted behavior changes?

**Comment by torley-gallium** [comment]
@n8henrie Sorry for the delay, just saw this. I appreciate all your care and investigation. Is there a new debug version to try? 2.4.2 still repros the issue for me (haven't deleted anything else).

**Comment by n8henrie** [comment]
@aberkvam @torley-gallium can you please try this and report back: https://github.com/quicksilver/Quicksilver/issues/2963#issuecomment-1912822642

**Comment by torley-gallium** [comment]
@n8henrie Thanks for checking in, so far so good with 2.4.2 . I don't see the enlarged text.
